packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/tera-optimize-ciemss-drilldown.vue

```diff
@@ -1079,23 +1084,6 @@ watch(
 </script>
 
 <style scoped>
-/* Left sidebar styles */
-:deep(.slider-content) {
-	background-color: var(--surface-100);
-	border-right: 1px solid var(--surface-border-light);
-	height: auto;
-	padding-bottom: 7rem;
```

Comment:
This bottom padding isn't being applied anymore

Reply:
Actually, you could probably add this bottom padding to the slider component, it's good to have on all of them so the content doesn't sit too close to the bottom of the panel